framework/Makefile

```diff
@@ -37,6 +37,7 @@ all: build
 
 install:
 	# SHARE
+	$(INSTALL_DIR) $(SHARE_INSTALLDIR)/bin
```

Comment:
```suggestion
	$(INSTALL_DIR) $(BIN_INSTALLDIR)
```

Reply:
Changed in b5a5da997b99742d75667000bfec999dd20878f8.